lib/src/views/Calendar/CalendarView.tsx

```diff
@@ -146,6 +72,9 @@ export const useStyles = makeStyles(
   { name: 'MuiPickersCalendarView' }
 );
 
+export const defaultReduceAnimations =
+  typeof window !== 'undefined' && /(android)/i.test(window.navigator.userAgent);
```

Comment:
For comparison in the core

```jsx
const disableSwipeToOpenDefault =
  typeof navigator !== 'undefined' && /iPad|iPhone|iPod/.test(navigator.userAgent);

```
```suggestion
  typeof navigator !== 'undefined' && /(android)/i.test(navigator.userAgent);
```